src/popover/src/Popover.js

```diff
@@ -372,11 +374,11 @@ export default class Popover extends Component {
               getRef(ref)
             }}
             data-state={state}
-            css={css}
             display={display}
             minWidth={minWidth}
             minHeight={minHeight}
             {...statelessProps}
+            className={cx(statelessProps.className, gcss(css))}
```

Comment:
This might produce `css-nil` class? Do we want `css ? gcss(css) : undefined`?

Reply:
I look at it

Reply:
fixed